tests/core/test_batch_related_objects.py

```diff
@@ -144,3 +148,28 @@ def test_RuntimeDataBatchSpec():
             "batch_data": "we don't check types yet",
         }
     )
+
+
+def test_runtime_batch_request_deepcopy():
```

Comment:
@roblim @jcampbell A brief docstring would be helpful here explaining that the memory location (returned by Python's internal `id()` method) is used to ascertain "sameness" and the lack thereof different objects.  Thanks.

Reply:
Addressed in dce22c4efc5b3775c87745bc707d13558f9297c9